lib/ruby_terraform/command_line/options.rb

```diff
@@ -14,11 +14,16 @@ def initialize(option_values:, command_arguments:)
         append_flags(command_arguments[:flags])
       end
 
+      def self.global_options
+        %i[chdir]
+      end
```

Comment:
Generally global things worry me :). I wonder if this could be a mixin into each command instead. That way, if a command came along that didn't support `chdir` we wouldn't have a broken contract. It's cheap to mix in something but gives you control to not mix it in in some cases.

Reply:
I was working on the basis that [terraform says its global](https://www.terraform.io/docs/cli/commands/index.html#switching-working-directory-with-chdir) but happy to move it to a mixin if that's preferred..